package.json

```diff
@@ -15,11 +15,18 @@
     "test": "npm run test:unit && npm run test:e2e"
   },
   "dependencies": {
+    "@auth0/auth0-spa-js": "^1.14.0",
     "@vue/babel-preset-app": "^4.5.10",
+    "axios": "^0.21.1",
+    "axios-request-handler": "^1.0.4",
+    "codecov": "^3.7.0",
     "compression-webpack-plugin": "^6.0.4",
+    "flush-promises": "^1.0.2",
     "idb": "^6.0.0",
     "konva": "^7.1.4",
     "moment": "^2.26.0",
+    "moment-range": "^4.0.2",
+    "moxios": "^0.4.0",
```

Comment:
`Moxios` is specially used to `intercept` Axios requests within the test suites for testing. 

If you are strongly in support of the browser `fetch` over `axios`, I would have to look into using jest mock() and remove `moxios`. 

However, it would require some significant refactoring of the tests. Would you prefer it comes in a separate PR?

Reply:
Fetch has now been switched to in the latest commits.